pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -103,6 +105,34 @@ var _ = Describe("Node-labeller config", func() {
 		Expect(cpuFeatures).To(HaveLen(4), "number of features doesn't match")
 	})
 
+	It("Should return the cpu features on s390x even without policy='require' property", func() {
+		if runtime.GOARCH != "s390x" {
+			Skip("Only relevant on s390x")
+		}
```

Comment:
But this will pass on x86 as well or not?

Reply:
It should not, since i have gated the code to only work on s390x

Reply:
No, as then it would fail on the other arches, since it would not default to `IBM` as CPU vendor, nor would it admit cpu features despite the `policy='require'` missing.

Reply:
Ah sorry, I now see. I would recommend embedding the arch to the node-labeller and then the tests can be independent from the env...